decision/policies.py

```diff
@@ -26,7 +27,9 @@ def get_best_action(world: World):
             reward = new_state.do_action(action)
 
             # Estimate value of making this action, after performing it and calculating the time it takes to perform.
-            reward += estimate_reward(new_state, world.get_remaining_time())
+            reward += (DISCOUNT_RATE ** world.time) * estimate_reward(
+                new_state, world.get_remaining_time()
+            )
```

Comment:
world.time is in minutes, should be divided by 60 to get it in hours. Else the discount rate will get VERY small VERY quick